fivetran/connectors.tf

```diff
@@ -54,3 +52,26 @@ resource "fivetran_connector" "github" {
     pat          = each.value.pat
   }
 }
+
+module "lambda_connector" {
+  for_each = { for lambda in var.sources_lambda : join("-", compact([lambda.service_name, lambda.project, lambda.environment])) => lambda }
+  source   = "../fivetran/connectors/lambda"
+
+  providers = { fivetran = fivetran }
```

Comment:
Do we also have to pass in a AWS provider if we want to deploy lambdas into multiple regions? E.g. see http://go/infra-fivetran-multi-region-lambda

Reply:
How about just passing the AWS_REGION as env variables?
https://github.com/dbl-works/terraform/pull/137/files#diff-b6ed8ac68eaaeb7c15e9437b253a9c87749872d3f796d438e53703f69d0f2e02R91

Reply:
if that works to deploy to multiple regions, sure

Reply:
I was thinking that we can passing the AWS_REGION as an env variables so the script can fetch the cloudwatch metrics from the region we set. However, it doesn't seems to be a good idea now because
1. AWS_REGION is an reserved env variables and cannot be overwritten.
```
InvalidParameterValueException: Lambda was unable to configure your environment variables because the environment variables you have provided contains reserved keys that are currently not supported for modification. Reserved keys used in this request: AWS_REGION
```
2. Even I manage to overwrite it by changing the env name, it also seems weird that a lambda function in US region can fetch data from EU cloudwatch (?)